docs/get-started/add-transformations.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_label: 'Add a transformation'
+sidebar_position: 4
+---
+
+# Add modules
+
+Following the importation of data sources we will now add the next elements in line: Holium **transformations**.
+
+Holium **transformations** are contained in **modules**, they can be considered as functions. Holium
+**modules** are compiled Wasm bytecodes that follows the [SDK specifications](../reference/sdk/specifications.md).
+
+In this tutorial we will be using **modules** compiled from Rust code that integrates the Rust SDK. For 
+more information on how it works please refer to the [dedicated reference](../reference/sdk/rust-sdk/usage.md).
+
+As we want to keep our DAG simple we will only add a single **transformation** in our pipeline DAG.
+
+The **module** code that we are using can be found in the [tutorial repository](https://github.com/polyphene/getting-started/tree/main/module/avocado_operations).
+A dedicated folder is also provided with [compiled assets](https://github.com/polyphene/getting-started/tree/main/assets/wasm).
+
+To create a **transformation** we can use the `holium transformation create` sub-command:
+
+```shell
+$ holium transformation create avocado_operations --bytecode avocado_operations.wasm --handle get_sales_by_year --json-schema-in '{\"type\": \"array\",\"prefixItems\": [{\"type\": \"array\",\"items\": {\"type\": \"object\",\"properties\": {\"plu_4046\": {\"type\": \"number\"},\"plu_4225\": {\"type\": \"number\"},\"plu_4770\": {\"type\": \"number\"},\"date\": {\"type\": \"string\"},\"average_price\": {\"type\": \"number\"},\"total_volume\": {\"type\": \"number\"},\"total_bags\": {\"type\": \"number\"},\"small_bags\": {\"type\": \"number\"},\"large_bags\": {\"type\": \"number\"},\"xlarge_bags\": {\"type\": \"number\"},\"avocado_type\": {\"type\": \"string\"},\"year\": {\"type\": \"number\"},\"geography\": {\"type\": \"string\"}}}}, {\"type\": \"number\"}]}' --json-schema-out '{\"type\":\"array\", \"prefixItems\":[{\"type\":\"array\", \"items\": {\"type\":\"array\", \"prefixItems\": [{\"type\": \"string\"}, {\"type\": \"number\"}]}}]}'
+new object created: avocado_operation
+```
+
+
+Once it is done we can list our available **transformation** by using the `holium transformation list`
+sub-command:
+
+```shell
+$ holium transformation list
+┌────────────────────┬───────────────────┬─────────────────┬──────────────────────────────┬──────────────────────────────┐
```

Comment:
Seems like a wrong output for the given example.

Reply:
Copied back all print command result from own CLI after a working running test in https://github.com/polyphene/docs.holium.org/pull/16/commits/054409ef8ae5bf0ac357484526e9e5038f708596